nbconvert/filters/markdown.py

```diff
@@ -8,18 +8,24 @@
 
 from __future__ import print_function
 import re
+import warnings
 
 try:
     from .markdown_mistune import markdown2html_mistune
 except ImportError as e:
     # store in variable for Python 3
     _mistune_import_error = e
 
+    warnings.warn(
+            "Conversion to other formats without mistune has been deprecated as of nbconvert 5.2."
+            "It has been required for html export since at least nbconvert 4.0.",
+            FutureWarning)
```

Comment:
Is it important to get rid of this possibility? As far as I'm aware, we don't need mistune for any other export formats. I don't think it's hurting us to leave this try/except here.